presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -505,3 +401,177 @@ exists as we've rollbacked to the previous state.
+     comment   | varchar |       |
+     zipcode   | varchar |       |
+    (5 rows)
+
+We can also rename the new column to `location`:
+
+.. code-block:: sql
+
+    ALTER TABLE ctas_nation RENAME COLUMN zipcode TO location;
+    DESCRIBE ctas_nation;
+
+.. code-block:: text
+
+      Column   |  Type   | Extra | Comment
+    -----------+---------+-------+---------
+     nationkey | bigint  |       |
+     name      | varchar |       |
+     regionkey | bigint  |       |
+     comment   | varchar |       |
+     location  | varchar |       |
+    (5 rows)
+
+Finally, we can delete the new column. The table columns will be restored to the
+original state.
+
+.. code-block:: sql
+
+    ALTER TABLE ctas_nation DROP COLUMN location;
+    DESCRIBE ctas_nation;
+
+.. code-block:: text
+
+      Column   |  Type   | Extra | Comment
+    -----------+---------+-------+---------
+     nationkey | bigint  |       |
+     name      | varchar |       |
+     regionkey | bigint  |       |
+     comment   | varchar |       |
+    (4 rows)
+
+Examples
+--------
+
+The Iceberg connector supports querying and manipulating Iceberg tables and schemas
+(databases). Here are the examples of SQL Operations supported from Presto :
+
+Create a schema
+^^^^^^^^^^^^^^^
+
+Create a new Iceberg schema named ``web`` that will store tables in an
+S3 bucket named ``my-bucket``::
+
+    CREATE SCHEMA iceberg.web
+    WITH (location = 's3://my-bucket/')
+
+Create table
+^^^^^^^^^^^^^
+
+Create a new Iceberg table named ``page_views`` in the ``web`` schema
+that is stored using the ORC file format, partitioned by date and
+country::
+
+    CREATE TABLE iceberg.web.page_views (
+      view_time timestamp,
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    )
+    WITH (
+      format = 'ORC',
+      partitioned_by = ARRAY['ds', 'country']
+    )
+
+Create a Iceberg table with Iceberg format version 2::
+
+    CREATE TABLE iceberg.web.page_views (
+      view_time timestamp,
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    )
+    WITH (
+      format = 'ORC',
+      partitioned_by = ARRAY['ds', 'country'],
+      format_version = '2'
+    )
+
+Create table as Select
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a new table ``page_views_new`` from an existing table ``page_views``::
+
+    CREATE TABLE iceberg.web.page_views_new AS SELECT * FROM iceberg.web.page_views
+
+Alter table
+^^^^^^^^^^^
+
+Alter table operations are supported in the connector::
+
+     ALTER TABLE iceberg.web.page_views ADD COLUMN zipcode VARCHAR;
+
+     ALTER TABLE iceberg.web.page_views RENAME COLUMN zipcode TO location;
+
+     ALTER TABLE iceberg.web.page_views DROP COLUMN location;
+
+Drop table
+^^^^^^^^^^^
+
+Drop the table ``page_views``. This only drops the metadata
+& data for the table::
+
+    DROP TABLE iceberg.web.page_views
+
+Drop schema
+^^^^^^^^^^^
+
+Drop a schema::
+
+    DROP SCHEMA iceberg.web
+
```

Comment:
I don't see the section for INSERT, DELETE, Merge or upsert? Let's have a section for it and state clearly what is supported and what is not.

Reply:
- I missed the INSERT example, will add that
- For DELETE I have mentioned in `Iceberg Connector Limitations` section that it's only supported if the ``WHERE`` clause matches entire partitions.
- MERGE & UPDATE sql syntax is not yet supported in Presto. I will open an issue for the same